repos/core-sdk.proj

```diff
@@ -80,8 +80,9 @@
 
     <!-- Produce stable versions for RTM only.  When https://github.com/dotnet/source-build/issues/550
                   is fixed we will be able to determine this automatically.  -->
-    <EnvironmentVariables Include="DropSuffix=true"
-                          Condition="'$(UseStableVersions)' == 'true'" />
+    <EnvironmentVariables Include="DropSuffix=true" />
```

Comment:
Do we want these to continue to be conditional based on a source-build property so we can specify stable versions in one place?

Reply:
Yes, reverted to keep the conditions.